.travis.yml

```diff
@@ -6,14 +6,16 @@ node_js:
     - "4.1"
 
 before_install:
+    - rvm install 2.2.2
```

Comment:
@dan-f I tried specifying two languages in my other PR but it doesn't work. I think that Travis must just pick the first one, and that happens to work for you.